odoo/tools/misc.py

```diff
@@ -1256,6 +1256,13 @@ def format_time(env, value, tz=False, time_format='medium', lang_code=False):
 
     return babel.dates.format_time(value, format=time_format, locale=locale)
 
+
+def _format_time_ago(env, time_delta, lang_code=False):
+    lang = env['res.lang']._lang_get(lang_code or env.context.get('lang') or 'en_US')
```

Comment:
`en_US` is a bad fallback as it may not be activated (you can create a db in a different language).
See #35633 where +/- the same problem was present.
`get_installed()` has the advantage to return you the code
(I know it is what it is done above but as we are introducing a new method ;-))

Reply:
Thanks ! I didn't know about res.lang.get_installed() ! Nice one too :D 
Well here as I just need the lang_code, I simplified a bit.